From 0dc58fba5380268908fc6445ecd4a79ceee68880 Mon Sep 17 00:00:00 2001
From: Masatake YAMATO <yamato@redhat.com>
Date: Fri, 8 Jun 2018 17:20:21 +0900
Subject: [PATCH] Tmain: initialize "V" variable explicitly

"V" variable was introduced as a placeholder that is used when I want
to run the test case under valgrind; the valgrind command line is set
to "V".

It seems that "Ubuntu PPA build" environment defines V as an environment
variable with value "1". See #1761.

As the result, the test case doesn't work well on the environment.

This commit clears the value for "V" to allow the test case works
on the environment.
---
 Tmain/parser-own-fields.d/run.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Tmain/parser-own-fields.d/run.sh b/Tmain/parser-own-fields.d/run.sh
index 2e80efa9..e6238cc9 100644
--- a/Tmain/parser-own-fields.d/run.sh
+++ b/Tmain/parser-own-fields.d/run.sh
@@ -4,7 +4,10 @@
 . ../utils.sh
 
 CTAGS=$1
+
+V=
 # V=valgrind
+
 echo '#' disabling fields
 ${V} ${CTAGS} --options=NONE --options=./unknown.ctags -o - input.unknown
 
-- 
2.17.0

